test/elixir/test/reshard_basic_test.exs

```diff
@@ -156,8 +156,8 @@ defmodule ReshardBasicTest do
     resp = Couch.get("/#{db}/_shards")
     assert resp.status_code == 200
     shards = resp.body["shards"]
-    assert node1 not in shards["00000000-7fffffff"]
-    assert node1 not in shards["80000000-ffffffff"]
+    # assert node1 not in shards["00000000-7fffffff"]
```

Comment:
@nickva I've had to comment out these two lines. Is the test still valid?

Reply:
Still valid. But maybe better to replace the checks with 

```
    assert node1 not in Map.get(shards, "00000000-7fffffff", [])
    assert node1 not in Map.get(shards, "80000000-ffffffff", [])
```

What that means is before we were checking that after those ranges were split, the copies from the original node and range were gone. With other nodes in that range on node2 and node3, shards[00..-7f..] was returning a list of [node2, node3]. Then with n=1,there are no copies in the those ranges on any nodes so `shards[...]` was failing. The fix is to do a `get` with a default value of `[]`, so it would work on both n=1 and n>1

Reply:
Awesome thanks @nickva. I've changed the tests.